I've found a bug when trying to use the `default_wait` value. It wasn't using a `datetime.delta` format so I encapsulted it with that and it fixed the problem.

Here's the error we were getting

```
2024-09-05 13:51:05,965 [ERROR] sarracenia.flow gather flowCallback plugin <bound method Scheduled.gather of <sarracenia.flowcb.scheduled.Scheduled object a
t 0x7f34eff24be0>> crashed: '<' not supported between instances of 'int' and 'datetime.timedelta'
2024-09-05 13:51:05,965 [DEBUG] sarracenia.flow gather details:
Traceback (most recent call last):
  File "/net/local/home/leblanca/.local/lib/python3.10/site-packages/sarracenia/flow/__init__.py", line 1133, in gather
    retval = p(self.o.batch-so_far)
  File "/net/local/home/leblanca/.local/lib/python3.10/site-packages/sarracenia/flowcb/scheduled/__init__.py", line 100, in gather
    self.wait_until_next()
  File "/net/local/home/leblanca/.local/lib/python3.10/site-packages/sarracenia/flowcb/scheduled/__init__.py", line 221, in wait_until_next
    self.wait_seconds(self.default_wait)
  File "/net/local/home/leblanca/.local/lib/python3.10/site-packages/sarracenia/flowcb/scheduled/__init__.py", line 140, in wait_seconds
    if sleepfor < nap:
TypeError: '<' not supported between instances of 'int' and 'datetime.timedelta'

```

--- a/sarracenia/flowcb/scheduled/__init__.py
+++ b/sarracenia/flowcb/scheduled/__init__.py
@@ -112,7 +112,7 @@ class Scheduled(FlowCB):
         self.first_interval=True
 
         if self.o.scheduled_interval <= 0 and not self.appointments:
-            logger.info( f"no scheduled_interval or appointments (combination of scheduled_hour and scheduled_minute) set defaulting to every {self.default_wait} seconds" )
+            logger.info( f"no scheduled_interval or appointments (combination of scheduled_hour and scheduled_minute) set defaulting to every {self.default_wait.seconds} seconds" )
 
     def gather(self,messageCountMax):
 

--- a/tests/sarracenia/flowcb/scheduled/__scheduled___test.py
+++ b/tests/sarracenia/flowcb/scheduled/__scheduled___test.py
@@ -72,6 +72,18 @@ def test_schedules():
 
      """
      options = build_options()
+     options.scheduled_hour= []
+     options.scheduled_minute= []
+     options.scheduled_time= []
+     me=sarracenia.flowcb.scheduled.Scheduled(options)
+     me.update_appointments(midnight)
+
+     logger.error( f"{me.appointments=}" )
+     assert( len(me.appointments) == 0 ) 
+
+
+     options = build_options()
+     options = build_options()
      options.scheduled_hour= [ '1' ]
      options.scheduled_minute= [ '1' ]
      me=sarracenia.flowcb.scheduled.Scheduled(options)

--- a/tests/sarracenia/flowcb/scheduled/__scheduled___test.py
+++ b/tests/sarracenia/flowcb/scheduled/__scheduled___test.py
@@ -22,7 +22,7 @@ def build_options():
      o.finalize()
      return o
 
-def test_broker_finalize():
+def test_schedules():
 
      options= build_options()
      options.scheduled_time=[ "12:40", "7:37" ]
@@ -96,8 +96,6 @@ def test_broker_finalize():
      me.update_appointments(midnight)
      assert( len(me.appointments) == 2 ) 
 
-     logger.error( f"HOHO" )
-
      options= build_options()
      options.scheduled_hour= [ "2", "4" ]
      options.scheduled_minute= [ "2" ]
@@ -106,6 +104,30 @@ def test_broker_finalize():
      me.update_appointments(midnight)
 
      assert( len(me.appointments) == 4 ) 
+
+     """
+       expect appointments at 2:00, and 4:00
+     """
+     options= build_options()
+     options.scheduled_hour= [ "2", "4" ]
+     options.scheduled_minute= [ ]
+     options.scheduled_time=[ ]
+     me=sarracenia.flowcb.scheduled.Scheduled(options)
+     me.update_appointments(midnight)
+
+     assert( len(me.appointments) == 2 ) 
+
+     """
+       expect appointments at :02, and :04 every hour.
+     """
+     options= build_options()
+     options.scheduled_minute= [ "2", "4" ]
+     options.scheduled_hour= [ ]
+     options.scheduled_time=[ ]
+     me=sarracenia.flowcb.scheduled.Scheduled(options)
+     me.update_appointments(midnight)
+
+     assert( len(me.appointments) == 48 ) 
 
      #logger.error( f" {me.appointments=} {len(me.appointments)=}" )
      #logger.error( f" HOHO {len(me.appointments)=}" )

